.gitignore

```diff
@@ -0,0 +1,3 @@
+/target
+.idea/
```

Comment:
`.idea\` should not be part of the gitingore of the individual repository, but rather can be defined on the devs machine as part of the global gitignore on the system level. The reason is that different people will use different IDEs (or no IDEs at all - hardcore), and gitignore would be polluted this way.

Reply:
Removed